verify/verify-golint.sh

```diff
@@ -22,7 +22,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
 GO_VERSION=($(go version))
 # golint only works for golang 1.5+
-if [[ -n $(echo "${GO_VERSION[2]}" | grep -E 'go1.1|go1.2|go1.3|go1.4') ]]; then
+if [[ -n $(echo "${GO_VERSION[2]}" | grep -E '(go1.1|go1.2|go1.3|go1.4)(\.|$)') ]]; then
```

Comment:
Let's remove this check completely instead.
We're already on much higher go version everywhere.

Reply:
done